src/fights.ts

```diff
@@ -1406,7 +1406,7 @@ const freeFightSources = [
       return false; // We have to spend turns to get the artifact or kill the boss
     },
     () => {
-      if (have($item`Rufus's shadow lodestone`)) setChoice(1500, 2);
+      setChoice(1500, 2);
```

Comment:
i think this looks fine-ish but i'm not sure i understand why exactly the have needs to be removed here? choice 1500 is the lodestone choice, there shouldn't possibly be a time where the user enters a shadow rift and encounters 1500 without the have being true

Reply:
It's because we have multiple adv1s in this single codeblock; you might not start with a lodestone, but that doesn't mean you won't have one later